minfraud/request.py

```diff
@@ -91,29 +282,43 @@ def maybe_hash_email(transaction):
     if address is None:
         return
 
-    address = address.lower().strip()
-
-    at_idx = address.rfind("@")
-    if at_idx == -1:
+    address = _clean_email(address)
+    if not address:
         return
 
-    domain = _clean_domain(address[at_idx + 1 :])  # noqa
-    local_part = address[:at_idx]
-
+    domain = address.split("@")[1]
```

Comment:
Perhaps we should return the two pieces from `_clean_email` if we need to break it up immediately after anyway. I think technically we should be accounting for multiple `@` for instance (which that function would handle I believe).